README.rst

```diff
@@ -165,10 +165,10 @@ Add the following settings to ``settings.py``:
 
```

Comment:
In README, `Python 2.7 or Python >= 3.5` needs to be updated! :smile:

Reply:
Updated it